app/javascript/components/editor/TestRunSummary.tsx

```diff
@@ -198,6 +198,9 @@ const TestRunSummaryContent = ({
         </div>
       )
     }
+    case TestRunStatus.CANCELLED: {
```

Comment:
I added this. Do we want a state to show to the user that they've cancelled their test?